src/components/content/deployedServices/myServices/MyServices.tsx

```diff
@@ -1011,6 +1175,22 @@ function MyServices(): React.JSX.Element {
                 </Modal>
             ) : null}
 
+            {activeRecord ? (
+                <Modal
+                    open={isLocksModalOpen}
+                    title={locksTitle()}
```

Comment:
title is misplaced. It has more space than the form elements. Form elements must be located inside the form title. 


![image](https://github.com/eclipse-xpanse/xpanse-ui/assets/54129659/08644549-af89-4fff-afda-75433e1657a8)